README.md

```diff
@@ -69,6 +69,7 @@ If you have a suggestion for an app, please post in the [BigBearCommunity](https
 | Mailpit                             | axllent/mailpit                                  | v1.13                    | [YouTube Video](https://youtu.be/2MY3S6csrVw)                                                                       |
 | Mealie                              | hkotel/mealie                                    | v1.2.0                   | [YouTube Video](https://youtu.be/S4MfNLV2Uf4)                                                                       |                                                                                                |
 | MIND                                | mrcas/mind                                       | v1.3.3                   |                                                                                                                     |
+| Minio                               | minio/minio                                      | latest              |                                                                                                                     |
```

Comment:
The version needs to be updated: `RELEASE.2024-02-24T17-11-14Z`

Reply:
done :-)